.circleci/config.yml

```diff
@@ -4,17 +4,17 @@ orbs:
   silta: silta/silta@0.1
 
 executors:
-  cicd74:
+  cicd80:
     docker:
-      - image: eu.gcr.io/silta-images/cicd:circleci-php7.4-node14-composer2-v0.1
+      - image: eu.gcr.io/silta-images/cicd:circleci-php8.0-node14-composer2-v0.1
```

Comment:
Would it make sense to update node version also since such image exists: https://eu.gcr.io/silta-images/cicd:circleci-php8.0-node16-composer2-v0.1

Reply:
Node 14 is the current default in Lando. Switching to node 16 would require update to .lando.yml as well.

Reply:
Good point, following for `.lando.yml` line 88-89 should do it:
```
  node:
    type: node:16
```

Reply:
Yes .. @tormi's call as this project is closely followed recommendations from upstreams (e.g. core & lando).

Reply:
We've been using non-default Node versions before. I'll push the v14 > v16 change.